public/data/crazydevlegend.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Robert Carden",
+  "type": "personal",
+  "bio": "Full-stack | blockchain expert | Open Source Enthusiast!",
```

I recommend capitalizing the `b` here. What do you think?
```suggestion
  "bio": "Full-stack | Blockchain expert | Open Source Enthusiast!",
```

---

Yes, agreed, @Panquesito7 
Just fixed it and now ready to merge...